### PR TITLE
Added new color variables and updated hex values in the documentation

### PR DESCRIPTION
**Pull request by emilybrick**
Notes are here:
https://docs.google.com/document/d/1bRNcUS9t_0WL_ckkMetWRu_jp-KIQOUByd-Rb903zy0/edit

Started working on this yesterday! Still have not done:
an accessibility pass, other than testing these on a crappy screen (new colors look better than old colors fwiw)

One thing to note: I am not using a new color for text-gray-lightest, as the new colors version of this was way too light to be readable. For more context, check out the doc above! 

I'm sure there are a bajillion mistakes! 🎱

### CODE REVIEW

**Comment by allisonchefec** [comment]
Thank you for doing this!! I'm having trouble with these new grays :-/

Fill-gray-lighter feels *really* dark! Here's a comparison: http://take.ms/QMhnP This is much more like our current middle gray fill (`fill-gray`) 

As you can see, the light grey background of Solid is now super dark:
screen shot 2016-11-02 at 6 19 34 pm

Here's how it looks on the homepage, comparing old and new: http://take.ms/8Mj6A

The new `fill-gray` seems a bit too close to `fill-gray-darker` : http://take.ms/kzHNP
screen shot 2016-11-02 at 6 18 49 pm Not sure if we'd have a use for both these dark greys.

I don't feel comfortable shipping these new light and mid grays. Do you agree? 

The other colors feel great to me.

**Comment by emilybrick** [comment]
yep it's quite dark! i figured we were going to have to compromise somewhere. i mentioned above about text-gray-lightest, probably the same pattern can follow for fill-gray-lighter. Unsure about the 'middle' gray. Will have to check that out tomorrow, going to stop doing work now :).

**Comment by emilybrick** [comment]
Fill Grays
fill grays now match all of our gray colors
the main difference between our grays and branding gray is that they have a light gray value between our fill-gray and fill-gray-lighter that's #e0e0e0, which felt **too dark** to become our fill-gray-lighter. So! The colors are in the branding sheet if need be, but will not be included the Solid colors for now.

Text Grays
Text gray lightest is the same value as before (#999999)\
Text-gray, I switched to #222 (same value as fill-gray-darker, and it's the "darkest-gray" in the branding sheet). The next value up is lighter than our current text-gray, so I am less in favor of that. 

I'm into this change because it feels less muddy, especially on weaker screens with lower resolution. Here is a compare of the homepage (darker gray on left).
text-gray-compare

Let's talk about reservations in the solid meeting!

**Comment by emilybrick** [comment]
Hey friends! This is ready to go, if anyone wants to take a last look! @allisonchefec @lindseymaratta 
Follow up card should include:

-adding teal
-adding purple
-darkening fill-gray-lightest a hair

File: _lib/solid-helpers/_variables.scss [hunk 1]
```diff
@@ -50,20 +50,20 @@ $z4: 400 !default;
...
-$fill-yellow-lighter: #fffab6 !default;
-$fill-green: #6fb824 !default;
-$fill-green-lighter: #f1f8e9 !default;
-$fill-orange: #f49c1a !default;
-$fill-promoted-orange: #edb802 !default;
-$fill-gray-lighter: #f4f4f4 !default;
-$fill-gray: #aaa !default;
-$fill-gray-darker: #222 !default;
-$fill-white: #fff !default;
+$fill-red: #e32;
```
Comment (allisonchefec) [inline _lib/solid-helpers/_variables.scss]: did we mean to remove the `!default` from all the variables here? if not, i think we should add em back in

**Comment by emilybrick** [comment]
Nope! Accident because the defaults were added after I started this branch. Good catch!

**Comment by emilybrick** [comment]
Adjusted - also fixed the vertical alignment of the !defaults in the variable scss file

**Comment by lindseymaratta** [comment]
The contrast for the green success messaging is pretty low. Definitely fails the WCAG but maybe if we used a `lighten` function for the background it could work? Could be addressed in a follow up or as part of the accessibility stuff.

https://monosnap.com/file/5RCLwJlcUmOCqglq840ppHpBCENlNC.png

Also still feeling some reservation about the `text-gray`s - seeing the `svg-fill` grays next to each other is what really makes it feel redundant, but it would be weird to not be consistent. Though if you're confident that the change is worth it, I trust ya!

https://monosnap.com/file/N4GxTpHjuJ8oEBKL0ZVcT4HNSNzlrm.png

Other than these concerns (do what you want with them!) and maybe a rebase to preview the integration with the last release, it lgtm. 👍

**Comment by allisonchefec** [comment]
Ah good catch @lindseymaratta. I know our svg fills are following our text colors, but in this case this does feel too dark, as did the other medium gray we tried, but then vetoed. 

Do we want to revert the updated `text-gray-lighter` color? Or do we want to just update this svg fill color to be something we're happy with? My vote would be to revert the updated text-gray-lighter color.

**Comment by lindseymaratta** [comment]
yeah I agree with reverting the updated `text-gray-lighter` - that way we can confidently pair icon and text of the same color

**Comment by emilybrick** [comment]
Latest push (maybe final...a little late in the game to be making big changes to this now): I reverted text-gray-lighter back to 666 which should fix the mid SVG gray as well

**Comment by allisonchefec** [comment]
cool! let's do it!

**Pull request merged**